ci/pingcap_tidb_operator_build_kind.groovy

```diff
@@ -48,54 +37,29 @@ def call(BUILD_BRANCH, CREDENTIALS_ID) {
 							sudo yum install -y mercurial
 						fi
 						hg --version
+						make check-setup
+						make check
+						make test
 						make
-						rm -rf tests/images/test-apiserver/bin
 						make e2e-build
 						"""
 					}
 				}
-				stash excludes: "${PROJECT_DIR}/vendor/**", includes: "${PROJECT_DIR}/**", name: "tidb-operator"
+				stash excludes: "${PROJECT_DIR}/vendor/**,${PROJECT_DIR}/deploy/**", includes: "${PROJECT_DIR}/**", name: "tidb-operator"
```

Comment:
terraform modules is very large, so we should skip them here to speed up this process